### PR TITLE
[Configuration] Change default jobs

### PR DESCRIPTION
**Pull request by lbarcziova**
Update in default values in job config (packit-service/packit#703).

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ jobs:
 If there is no `jobs` section in the configuration file, jobs default to:
 ```
 jobs:
-- job: copr_build
+- job: tests
```
Comment (lachmanfrantisek) [inline content/docs/configuration.md]: Thanks, I forgot to do it...

**Comment by lbarcziova** [comment]
> Looks like the regeneration did not go well...

I do not know what's the problem, I did it as always..

**Comment by TomasTomecek** [comment]
I am using older hugo, so I regen'd and rebased your PR.

**Comment by lbarcziova** [comment]
Thanks

**Pull request merged**